src/main/java/yanagishima/servlet/CommentServlet.java

```diff
@@ -34,20 +32,16 @@
     private final YanagishimaConfig config;
     private final TinyOrm db;
 
+    @DatasourceAuth
     @PostMapping("comment")
     public Map<String, Object> post(@RequestParam String datasource,
                                     @RequestParam String engine,
                                     @RequestParam(name = "queryid") String queryId,
                                     @RequestParam Optional<Integer> like,
                                     @RequestParam(required = false) String content,
-                                    HttpServletRequest request, HttpServletResponse response) {
+                                    HttpServletRequest request) {
         Map<String, Object> responseBody = new HashMap<>();
         try {
-            if (config.isCheckDatasource() && !validateDatasource(request, datasource)) {
-                sendForbiddenError(response);
-                return responseBody;
```

Comment:
The original logic is changed?

Reply:
No. User will get forbidden error at L47 and return at L48 is basically meaningless.